CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# How to become a contributor and submit your own code
+
+## Contributor License Agreements
```

Comment:
@danisla Since this is not a Google repo, I believe we're leaving off the CLA bits.

Reply:
Good point, what about the license header in files?

Reply:
I've still been keeping that since the individual files are copyright Google.